test/export/test_export.py

```diff
@@ -3854,6 +3858,22 @@ def foo_unbacked_fake_impl(a, b):
         self.assertEqual(type(path[2]).__name__, "DivideByKey")
         self.assertEqual(path[2].divisor, 10)
 
+        # collect bound symbols
+        bound = set()
+        for node in ep.graph.nodes:
+            bound.update(node.meta.get("unbacked_bindings", {}))
+
+        # check ShapeEnv counters compared to binding indices
+        shape_env = detect_fake_mode(
```

Comment:
I think there's another util to directly detect fake mode from a graph module / ep, which is also used elsewhere in our code. Maybe switch to that?